### PR TITLE
Pull request for mclass package

### PR DESCRIPTION
**Pull request by edwinm**
I could add more files in my own jsdelvr clone instead of jsdelivr/jsdelivr, was not clear from documentation. Sorry for confusion. Thanks.

### CODE REVIEW

**Comment by jimaek** [comment]
The files must be located inside /files and not in root.

@megawac We should make the bot check for that as well

File: mclass/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "mclass",
+  "repo": "edwinm/mClass",
+  "files": {
+    "include": ["mclass.js"]
```
Comment (jimaek) [inline mclass/update.json]: The file must be minified

**Comment by megawac** [comment]
I could easily add warning (it checks files) I avoided it originally in case we wanted to add files like `Contributing.md`, etc to `/`

**Comment by edwinm** [comment]
Added minified file and configuration files in right directory.

File: files/mclass/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "mclass",
+  "repo": "edwinm/mClass",
+  "files": {
+    "include": ["mclass-min.js"]
```
Comment (jimaek) [inline files/mclass/update.json]: This file is not present here https://github.com/edwinm/mClass/tree/1.0.1
The bot will fail

**Comment by edwinm** [comment]
The tag is fixed now. Please retry. Sorry for inconvenience.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**